proposals/lip-properties-serialization-and-initial-values-of-the-interoperability-module.md

```diff
@@ -0,0 +1,862 @@
+
+CCUs are used to post the updated state of the sending chain on the receiving chain. Furthermore, they transmit the cross-chain messages that need to be sent to the receiving chain (see Figure 1). We introduce two different CCUs, one for posting on the Lisk mainchain, and the other for posting on sidechains. They differ in the way the included messages are handled: on mainchain, messages targeting another sidechain are forwarded to that sidechain outbox, while messages targeting the mainchain are simply processed. On the other hand, a CCU posted on a sidechain can only contain CCMs targeting that sidechain, being invalid otherwise.
+
+
+#### [Recovery Initialization Command][recovery-LIP]
+This command is used to initialize a terminated chain account on a sidechain. The user proves with an inclusion proof that the target chain state on mainchain implies that that chain is terminated (either the status is set to 'terminated' or the liveness condition is violated). Once the terminated chain account has been initialized, the state recovery command can be issued in the sidechain.
+
+
+#### [State Recovery Command][recovery-LIP]
+
+This command is used to recover a certain state (for example fungible and non-fungible tokens) from a sidechain that has been terminated. The user proves the existence of an entry in the terminated chain state with an inclusion proof.
+The proof is validated against the current state root stored in the terminated chain account. 
+The recovery of the state is then handled by the relevant module (for example the token module would refund the user).
+
+
+#### [Message Recovery Command][recovery-LIP]
+
+This command is used to recover a pending message in the outbox of a terminated chain. The user proves with an inclusion proof that the message is in the terminated sidechain outbox. The proof is validated against the outbox root stored in the sidechain account on mainchain. The recovered message is then bounced back to the original sending chain (which could be the mainchain).
+
+
+### Liveness Condition
+
+Active sidechains are required to prove their liveness to the mainchain at least once every 30 days. This is done by including a CCU on the mainchain. If a sidechain does not post a CCU within 30 days, the sidechain account is terminated.  
+
+This rule guarantees that users do not send funds to inactive sidechains and that users who have tokens in a sidechain which stops communicating with the ecosystem can recover their tokens. The same rule applies to the mainchain account on a sidechain. At least once within 30 days, a mainchain CCU has to be included in the sidechain, otherwise, the mainchain account on the sidechain is terminated.
+
+
+### Life Cycle of a Sidechain
+
+The life cycle of a sidechain can be split into 3 parts, corresponding to the 3 [values of the account status property](#List-of-Account-status-Values): ''registered'', ''active'', and ''terminated''. 
```

Comment:
Seems like this link to the status is broken

Reply:
yep, we removed the section